common/src/main/java/org/jboss/hal/testsuite/finder/FinderNavigation.java

```diff
@@ -106,6 +109,17 @@ public FinderNavigation(final WebDriver browser, final Class<? extends BasePage>
     }
 
     /**
+     * Sets the timeout in seconds for the application to be loaded in the beginning of the navigation after refresh.
+     * Use carefully in exceptional situations where default timeout of 30 seconds is not sufficient,
+     * e.g. for large domain performance tests.
+     * @param pageLoadTimeout
```

Comment:
Does it have any benefit to have here the `@param pageLoadTimeout` without description? => Give there short description that it represents timeout in seconds even though you can guess it from general description.